src/app/TargetView/TargetContextSelector.tsx

```diff
@@ -119,110 +123,102 @@ export const TargetContextSelector: React.FC<TargetContextSelectorProps> = ({ cl
   const selectOptions = React.useMemo(() => {
     if (noOptions) {
       return [
-        <SelectOption key={'no-target-found'} isDisabled>
+        <DropdownItem itemId={undefined} key={'no-target-found'} isDisabled>
           No target found
-        </SelectOption>,
+        </DropdownItem>,
       ];
     }
 
-    const favSet = new Set(favorites);
+    const matchExp = new RegExp(searchTerm, 'i');
+    const filteredTargets = targets.filter((t) =>
+      [t.alias, t.connectUrl, getAnnotation(t.annotations.cryostat, 'REALM') ?? ''].some((v) => matchExp.test(v)),
```

Comment:
Alias and group search works perfectly. But when I tried with the URL, something seems off.

![image](https://github.com/user-attachments/assets/73cdd2ab-ba96-4f77-90c1-eb08bd835918)

Reply:
[See](https://github.com/cryostatio/cryostat-web/pull/1304#discussion_r1692110136)

Reply:
Nice, I had only been testing using `smoketest.bash -Ok`, so I didn't have enough targets here to really test things.